modules/wmco-prerequisites.adoc

```diff
@@ -0,0 +1,86 @@
+// Module included in the following assemblies:
+//
+// * windows_containers/understanding-windows-container-workloads.adoc
+
+[id="wmco-prerequisites_{context}"]
+= Windows Machine Config Operator prerequisites
+
+The following information details the supported cloud provider versions, Windows Server versions, and networking configurations for the Windows Machine Config Operator. See the vSphere documentation for any information that is relevant to only that platform.
+
+== Supported cloud providers based on {product-title} and WMCO versions
+
+[cols="3",options="header"]
+|===
+|Cloud provider
+|Supported {product-title} version
+|Supported WMCO version
+
+|Amazon Web Services (AWS)
+|4.6+
+|WMCO 1.0+
+
+|Microsoft Azure
+|4.6+
+|WMCO 1.0+
+
+|VMware vSphere
+|4.7+
+|WMCO 2.0+
+|===
+
+## Supported Windows Server versions
+
+The following table lists the supported link:https://docs.microsoft.com/en-us/windows/release-health/release-information[Windows Server version] based on the applicable cloud provider. Any unlisted Windows Server version are NOT supported and will cause errors. To prevent these errors, only use the appropriate version according to the cloud provider in use.
+
+[cols="2",options="header"]
+|===
+|Cloud provider
+|Supported Windows Server version
+
+|Amazon Web Services (AWS)
+|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809
+
+|Microsoft Azure
+|Windows Server Long-Term Servicing Channel (LTSC): Windows Server 1809
+
+|VMware vSphere
+|Windows Server Semi-Annual Channel (SAC): Windows Server 2004
+|===
+
+== Supported networking
+OVN-Kubernetes hybrid networking is the only supported networking configuration. See the additional resources below for more information on this functionality. The following tables outline the type of networking configuration and Windows Server versions to used based on your cloud provider. You must specify the network configuration when you install the cluster. However, OpenShift SDN networking is the default network for {product-title} clusters. However, OpenShift SDN is not supported by WMCO.
```

Comment:
Add a blank line between the heading and the para.  I'm having trouble spotting an explicit rule, but it is the convention in the repository.

Reply:
There was a refresh in the midst of the review.  Please look at these comments:

It's nitpicky, but please use "Hybrid networking with OVN-Kubernetes is the only..." That is the description that has been used in the networking section of the repository.

?: s/to used/to use/

Maybe change the first "However" to "Be aware that OpenShift SDN networking..."